app/src/main/java/com/android/shelfLife/viewmodel/overview/IndividualFoodItemViewModel.kt

```diff
@@ -0,0 +1,29 @@
+package com.android.shelfLife.viewmodel.overview
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import com.android.shelfLife.model.newFoodItem.FoodItem
+import com.android.shelfLife.model.newFoodItem.FoodItemRepository
+import com.android.shelfLife.model.user.UserRepository
+
+class IndividualFoodItemViewModel (
+    private val foodItemRepository: FoodItemRepository,
+            private val userRepository: UserRepository
+): ViewModel(){
+    var selectedFood by mutableStateOf<FoodItem?>(null)
+
+    init{
+        selectedFood = foodItemRepository.selectedFoodItem.value
+    }
+
```

Comment:
there is no selectFoodItem function as it is not necessary for this viewModel. The only thing that are possible on this screen is to read the information from the selected foodItem and delete the selectedFoodItem.